src/index.html

```diff
@@ -7,10 +7,35 @@
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
+    <title>Search bar airbnb</title>
   </head>
   <body>
     <h1>Search bar airbnb</h1>
+    <form 
+    action="/" 
+    method="post"
+    data-qa="big">
+      <label>
+        <input
+        data-qa="keypress"
+        type="search" 
+        name="search" 
+        class="largeInput"
```

Comment:
```suggestion
        class="input largeInput"
```
в тебе дублюються стилі для largeInput та smallInput. Створи клас input, де пропиши загальні для обох інпутів стилі, а для largeInput та smallInput прописуй окремі стилі, які стосуються вже конкретно кожного інпута. І вкиористовуй ццих два класу разом, типу `class="input smallInput"`

Reply:
Я исправил насколько смог понять.